packages/react-component-library/src/components/Button/Button.tsx

```diff
@@ -2,6 +2,7 @@ import React from 'react'
 import classNames from 'classnames'
 
 export interface ButtonProps extends ComponentWithClass {
+  children?: string
```

Comment:
Dont need this

Reply:
You get a type error without this.

Reply:
thats weird. Component with class inherits from component which has children in it